src/module/applications/apps/power-roll-dialog.mjs

```diff
@@ -95,7 +95,7 @@ export default class PowerRollDialog extends RollDialog {
     context.damageOptions = null;
     for (const tier of PowerRoll.TIER_NAMES) {
 
-      const effect = context.ability.system.power.effects.getByType("damage").find(e => e.damage[tier].types.size > 1);
+      const effect = context.ability.system.power.effects.documentsByType.damage.find(e => e.damage[tier].types.size > 1);
```

Comment:
The `getByType` method, initially copied from dnd5e, has been yote as well, in favor of the now general `Collection#documentsByType` getter which is found on all EmbeddedCollections throughout the platform now.

No deprecation or shim has been put in place for `getByType`.